src/includes/javascript-crons-upsert.mdx

```diff
@@ -0,0 +1,124 @@
+You can create and update your Monitors programmatically with code rather than [creating and configuring them in Sentry.io](https://sentry.io/crons/create/).
+
+Using `Sentry.withMonitor()`, you can pass a third parameter with your monitor configuration:
+
+```javascript
+Sentry.withMonitor(
+  "<monitor-slug>",
+  () => {
+    // Execute your scheduled task here...
+  },
+  {
+    schedule: {
+      type: "crontab",
+      value: "* * * * *",
+    },
+    checkinMargin: 2, // In minutes. Optional.
+    maxRuntime: 10, // In minutes. Optional.
+    timezone: "America/Los_Angeles", // Optional.
+  }
```

Comment:
Would it make sense to define monitorConfig separately and then pass it in, as done in the next code sample? I think this would make it clearer what this sample is showing.

Reply:
Done